pwiz_tools/commandline/msconvert-help.txt

```diff
@@ -118,78 +118,42 @@ Options:
 Spectrum List Filters
 =====================
 
-Note: Filters are applied sequentially in the order that you list them, and the
-sequence order can make a large difference in your output.  In particular, the
-peakPicking filter must be first in line if you wish to use the vendor-supplied
-centroiding algorithms since these use the vendor DLLs, which only operate on
-raw untransformed data.
-
-Many filters take 'int_set' arguments.  An 'int_set' is a list of intervals of
-the form [a,b] or a[-][b].
-For example '[0,3]' and '0-3' both mean 'the set of integers from 0 to 3
-inclusive'.
+Note: Filters are applied sequentially in the order that you list them, and the sequence order can make a large difference in your output.  In particular, the peakPicking filter must be first in line if you wish to use the vendor-supplied centroiding algorithms since these use the vendor DLLs, which only operate on raw untransformed data.
+
+Many filters take 'int_set' arguments.  An 'int_set' is a list of intervals of the form [a,b] or a[-][b].
+For example '[0,3]' and '0-3' both mean 'the set of integers from 0 to 3 inclusive'.
 '1-' means 'the set of integers from 1 to the largest allowable number'.  
```

Comment:
Yes, this is much better, thanks!

Reply:
@chambm OK, I'll go ahead and merge if you're happy with it, and leave it to you to update
https://proteowizard.sourceforge.io/tools/msconvert.html
so that it functions mostly as a place to point users at
https://raw.githubusercontent.com/ProteoWizard/pwiz/master/pwiz_tools/commandline/msconvert-help.txt
(possibly with an iframe, up to you)